.github/workflows/deploy-environment.yml

```diff
@@ -1,13 +1,15 @@
 name: Deploy Argilla environment
 
 on:
+  workflow_dispatch:
```

Comment:
So with this you can click on run and manually create a pr environment for the PR on gcloud?